### PR TITLE
Initial Import of the yeoman generator

### PR DESCRIPTION
**Pull request by benoitf**
Yeoman Generator allowing to generate a frontend or backend plugin.

### Build:
```
$ yarn build
```

### run:
```
$ yo theia-plugin
```

### CODE REVIEW

**Review by evidolob — APPROVED** [review]

File: src/app/index.ts [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+
+    }
+
+    async doPrompt() {
+
+        if (!(this.options as any).pluginName) {
+            const answers = await this.prompt([{
+                type: 'input',
+                name: 'name',
+                message: "What is the plugin's name",
```
Comment (akurinnoy) [inline src/app/index.ts]: It seems question mark is missed at the end of this message.

Comment (benoitf) [reply src/app/index.ts]: fixed, thanks

File: templates/webpack.config.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+        libraryTarget: "var",
+        library: "simplePlugin",
+        <%} %>        
+        path: path.resolve(__dirname, 'dist')
+    },
+    externals: {
+		"@wiptheia/plugin": "theia",
+		"@heia/plugin": "theia"
+	}
+};
```
Comment (akurinnoy) [inline templates/webpack.config.js]: There are some EOLs missed.

**Review by akurinnoy — APPROVED** [review]

File: templates/webpack.config.js (hunk 2 above)
Comment (benoitf) [reply templates/webpack.config.js]: thanks, I think I fixed all EOF issues

**Pull request merged**